Lib/test/test_uuid.py

```diff
@@ -527,21 +529,22 @@ def check_node(self, node, requires=None, *, random=False):
         # `test_random_getnode()` method specifically.  Another case is the
         # Travis-CI case, which apparently only has locally administered MAC
         # addresses.
-        if not random and not os.getenv('TRAVIS'):
+        if not local and not os.getenv('TRAVIS'):
             self.assertFalse(node & (1 << 41), '%012x' % node)
+        is_multicast = (node & (1 << 40))
```

Comment:
We shouldn't test the multicast bit for `_unix_getnode()` and `_windll_getnode()`. Actually we should test it only for `_random_getnode()`, because the other methods can return an address with this bit set or clear (your original issue).